backend/src/akvo/flow_services/geoshape_export.clj

```diff
@@ -81,23 +116,44 @@
     (.mkdirs dir)
     (io/file dir filename)))
 
-(defn export [app-id form-id geoshape-question-id]
+(defn fetch-instance-data [server-base api-key instance-id]
+  (dissoc (bean (. (BulkDataServiceClient/fetchInstanceData instance-id server-base api-key) surveyInstanceData))
```

Comment:
The other `fetch-*` operations use _Remote API_. I think we should use that instead of keep using the _old_ way of accessing data, because:

* It's more efficient - we control the amount of entities we fetch per request
* We stop using the _pseudo-deprecated_ internal API, based on apiKey

Reply:
As per conversation, neither the internal API nor the remote API are the correct solution, so we will keep using the internal API until we decide for a proper solution (flow api maybe??))